### DISCUSSION TITLE
EPS?

### DISCUSSION THREAD

**Discussion by alekxvargas** [ccseer/Seer#395] [General]
Hi! Is there any EPS plugin available? I've notices that seer does not preview eps, even if they're pdf enabled. It would be cool, to have a wider support for design formats

**Comment by ccseer** [comment]
![image](https://user-images.githubusercontent.com/15963166/214385968-7b4a991f-de64-4eef-a137-bae60778c30f.png)

**Reply by ccseer** [reply]
https://github.com/ccseer/Seer/issues/386#issuecomment-1384704259
